app/src/main/java/com/example/antitheft/structure/GPSLocation.java

```diff
@@ -0,0 +1,49 @@
+package com.example.antitheft.structure;
+
+import android.location.Location;
+
+import com.eis.smslibrary.SMSPeer;
+
+/**
+ * //TODO
+ */
+public class GPSLocation {
+
+    Location location;
+    SMSPeer peer;
+
+    /**
+     * Constructor...//TODO
+     *
+     * @param peer
+     * @param location
+     */
+    GPSLocation(SMSPeer peer, Location location) {
+        //TODO
+    }
+
+    /**
+     * //TODO
+     */
+    public static Location getCurrentLocation() {
+        //TODO
+        return null;
+    }
+
+    /**
+     * //TODO
+     */
+    public SMSPeer getPeer() {
+        //TODO
+        return null;
+    }
+
+    /**
+     * //TODO
+     */
+    public GPSLocation getLocation() {
+        //TODO
+        return null;
+    }
```

Comment:
One question @AlbertoUrsino :
is this method correct, so I have to return the instance like a Singleton, or is it supposed to be
`public Location getLocation()`
?